dart/lib/src/protocol/sentry_span.dart

```diff
@@ -42,20 +42,20 @@ class SentrySpan extends ISentrySpan {
       return;
     }
 
-    final utcDateTime = getUtcDateTime();
-
     if (status != null) {
       _status = status;
     }
 
-    if (endTimestamp?.isBefore(_startTimestamp) ?? false) {
+    if (endTimestamp == null) {
+      _endTimestamp = _hub.options.clock().toUtc();
```

Comment:
Same as https://github.com/getsentry/sentry-dart/pull/1110/files#r1016301034
Double check all the calls to `_hub.options.clock()` since its already UTC

Reply:
same as above:

> since its already UTC

it is provided by the user, so there's no guarantee it is. Nor even option docs say it should be.

I can drop it of course, just doing the same that was already in this piece of the code.

Reply:
My point is not about making it UTC or not.
`_hub.options.clock()` already returns an UTC date, so there's no need to call `.toUtc()` again.